session4/src/com/itschool/homework/Challenge1.java

```diff
@@ -0,0 +1,27 @@
+package com.itschool.homework;
+
+import java.sql.SQLOutput;
```

Comment:
this is an unused import. I think you imported it by error. You can hover the mouse on it and you will see the suggestion to "Optimeze imports" (or if you are using the Eclipse Keymap - press CTRL + Shift + O on the keyboard) and the import will be removed.